src/fontra/views/editor/editor.js

```diff
@@ -692,25 +692,34 @@ export class EditorController {
     }
     this.basicContextMenuItems.push(MenuItemDivider);
 
+    const safariHTTPAgent =
+      navigator.userAgent.indexOf("Safari") > -1 &&
```

Comment:
In Chrome, the userAgent string also contains the substring "Safari".

Reply:
https://stackoverflow.com/questions/7944460/detect-safari-browser has some useful answers. I like this one:
```js
var isSafari = navigator.vendor && navigator.vendor.indexOf('Apple') > -1 &&
               navigator.userAgent &&
               navigator.userAgent.indexOf('CriOS') == -1 &&
               navigator.userAgent.indexOf('FxiOS') == -1;
```

Reply:
Maybe a combo?
```
const isSafari =
      navigator.vendor &&
      navigator.vendor.indexOf("Apple") > -1 &&
      navigator.userAgent.indexOf("Safari") > -1 &&
      navigator.userAgent.indexOf("CriOS") == -1 &&
      navigator.userAgent.indexOf("FxiOS") == -1 &&
      window.safari !== undefined &&
      window.location.protocol === "http:";
```

Reply:
Seems a bit redundant. Maybe only `window.safari !== undefined` and the `http:` check? I understand the former is only for desktop, which is fine for us actually.

Reply:
Yes, the `vendor` and` userAgent.indexOf `are probably going to give us some false positives.
Going back to the simple stuff!